spec/ics-007-tendermint-client/README.md

```diff
@@ -397,6 +409,10 @@ function verifyPacketReceiptAbsence(
     assert(clientState.latestHeight >= height)
     // check that the client is unfrozen or frozen at a higher height
     assert(clientState.frozenHeight === null || clientState.frozenHeight > height)
+    // fetch the processed time
+    processedTime = get("clients/{identifier}/processedTimes/{height}")
```

Comment:
Why do we need this check in packet receipt absence? I guess we only need the proof that the packet hasn't been received by the counterparty?

Reply:
This is a good question, I'm not sure. In the security model as I understand it, we want delays on all verification which could result in movement of funds, in order to allow time for evidence to be submitted, yes? Packet receipt absence does fall into this category, since it is used for proof-of-timeout which will e.g. refund funds to the sender of a fungible token transfer.

Reply:
I think we want to enforce the delay period specified in packet send on any packet-related verification. Since each step involves a race between the packet verification and misbehaviour submission that could invalidate the verification.

The original sender of the packet should be able to define a delay period that gives a bounded time for misbehaviour submission before packet-verification at every stage of packet lifecycle.